package.json

```diff
@@ -16,5 +16,10 @@
   "bugs": {
     "url": "https://github.com/team9353/food-deserts/issues"
   },
-  "homepage": "https://github.com/team9353/food-deserts#readme"
+  "homepage": "https://github.com/team9353/food-deserts#readme",
+  "dependencies": {
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "semantic-ui-react": "^0.88.2"
```

Comment:
I know this is a little confusing, but these dependencies should be added in the `client/package.json` file, not this one

Reply:
Ahh okay thanks!

Reply:
Looks like it's already in the client/package.json file too. I just deleted this from the main folder